src/java/org/apache/cassandra/metrics/ClientRequestMetrics.java

```diff
@@ -26,26 +26,44 @@
 import static org.apache.cassandra.metrics.CassandraMetricsRegistry.Metrics;
 
 
-public class ClientRequestMetrics extends LatencyMetrics
+public class ClientRequestMetrics
 {
     public final Meter timeouts;
     public final Meter unavailables;
     public final Meter failures;
 
-    public ClientRequestMetrics(String scope, String namePrefix)
-    {
-        super("ClientRequest", namePrefix, scope);
+    /**
+     * this is the metric that measures the actual execution time of a certain request;
+     * for example, the duration of StorageProxy::readRegular method for regular reads
+     */
+    public final LatencyMetrics executionTimeMetrics;
+
+    /**
+     * this is the metric that measures the time a request spent in the system;
+     * for example, the duration between queryStartNanoTime and the end of StorageProxy::readRegular method for regular reads
+     */
+    public final LatencyMetrics serviceTimeMetrics;
 
+    protected final String namePrefix;
+    protected final MetricNameFactory factory;
+
+    public ClientRequestMetrics(String scope, String prefix)
+    {
+        namePrefix = prefix;
+        factory = new DefaultNameFactory("ClientRequest", scope);
         timeouts = Metrics.meter(factory.createMetricName(namePrefix + "Timeouts"));
         unavailables = Metrics.meter(factory.createMetricName(namePrefix + "Unavailables"));
         failures = Metrics.meter(factory.createMetricName(namePrefix + "Failures"));
+        executionTimeMetrics = new LatencyMetrics(factory, namePrefix);
```

Comment:
this is supposed to retain backward compatibility wrt metric names of the previous version